definitions/divorce/json/CaseField.json

```diff
@@ -3228,5 +3228,13 @@
     "Label": "Final date to apply for Decree Absolute",
     "FieldType": "Date",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "27/08/2019",
+    "CaseTypeID": "DIVORCE",
+    "ID": "DecreeAbsoluteApplicationDate",
+    "Label": "Date when Petitioner or Respondent apply for Decree Absolute",
+    "FieldType": "Date",
```

Comment:
Can we have datetime here, if it's not too hard to do? Just in case we ever need it.